src/test/robot/api/storage/data/read_events.js

```diff
@@ -0,0 +1,34 @@
+var sharedConfig = {
+    "storage": {
+        "namespace": SCOPE_NAME
+    }
+}
+
+PreconditionLoad
+	.config(sharedConfig)
+	.config({
```

Comment:
`.config({})`
specify the count limit from the command line
this makes the scenario more independent and reusable

Reply:
Agree, done.

Reply:
where?